extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -510,6 +514,11 @@ export class TestObjectManagementService implements IObjectManagementService {
 					{ name: 'batch_mode_memory_grant_feedback', valueForPrimary: 'OFF', valueForSecondary: 'OFF' },
 					{ name: 'batch_mode_adaptive_joins', valueForPrimary: 'OFF', valueForSecondary: 'ON' },
 					{ name: 'tsql_scalar_udf_inlining', valueForPrimary: 'ON', valueForSecondary: 'ON' }
+				],
```

Comment:
Do we actually need this fake data still? Shouldn't we be using the real data by now?

Reply:
We use the real data from STS, this fake data is always helpful to load the dialogs when useTestService is set to true. With out this dialogs will be opened but are empty.

Reply:
Oh, I missed that this is in a TestObjectManagementService - nevermind then.